packages/pcd-collection/src/permissions.ts

```diff
@@ -0,0 +1,45 @@
+// TODO: test that PCDPermissionType and PCDActionType don't overlap
```

Comment:
could you add the test described in this todo? basically I want it to be impossible to assign a value in `PCDPermissionType` to a variable with a type `PCDActionType`

Reply:
I am not sure how to test this, since the TypeScript checker would disallow it and prevent it from compiling.

Reply:
Testing this by casting both enums to arrays of strings and checking there's no overlap.